Sources/StreamChat/Query/ChannelQuery_Tests.swift

```diff
@@ -57,6 +57,32 @@ class ChannelQuery_Tests: XCTestCase {
         let query2: ChannelQuery = .init(cid: cid)
         
         // Assert type and id are part of path
-        XCTAssertEqual(query2.pathParameters, "\(query2.type)/\(query2.id!)")
+        XCTAssertEqual(query2.pathParameters, "\(query2.type.rawValue)/\(query2.id!)")
+    }
+    
+    func test_pathParameters_with_customType() {
+        let query: ChannelQuery = .init(channelPayload: .init(
+            type: .custom("custom_type"),
+            name: .unique,
+            imageURL: .unique(),
+            team: nil,
+            members: [.unique],
+            invites: [],
+            extraData: .defaultValue
+        ))
+        XCTAssertEqual(query.pathParameters, "custom_type")
+    }
+    
+    func test_pathParameters_with_customType_and_id() {
```

Comment:
nit: we try to follow this pattern when naming tests

![](https://cdn-bdhbk.nitrocdn.com/RzEPfqBHiFVuBPFUnSCFvbLYsgnhmOov/assets/static/optimized/wp-content/uploads/2020/04/b4abc82287e1369b7287dd184f150940.TestNaming-big.png)

Reply:
👌